### ISSUE TITLE
Tracking the number of mobile terminals nearby

### ISSUE THREAD

**Issue by laabak84** [nootedandrooted/SDR-close-call-monitor-for-tetra#3]
Thanks for your great work. 
I am monitoring signals through openwebrx, and i can see signals on 382,2556 when a mobile terminal is nearby. I am not a signal expert, but i believe the number of lines i see in the waterfall, short bursts, are "ping" signals from the mobile termina. The closer the terminal is, the stronger the signal is. 

I have run your SDR-close-call-monitor-for-tetra, with 382e6 - 384e6. Meanwhile on another computer monitoring the signals on the same frequency. Unfortunately, the SDR-close-call-monitor-for-tetra does not register the signals.

I have tried the same with my PMR-radio on the 446 band, and the monitor immediately register the signal. Do you have any tips?

Love your work here! - PS, could it be the num_samples as the "ping" signals are very short?

**Comment by nootedandrooted** [comment]
Hi,

Thank you for your interest to my project. I suggest to use my other project's code called rtl-sdr-close-call-monitor for this purpose.

I haven't have had enough time to improve this specific project but the other project I suggested should work for Tetra signals as well. It's basically the same code as this but improved.

**Comment by laabak84** [comment]
Thanks for your reply! I actually started with the other repository before i found this. Still have the same problem :-( I can see the signals when terminals are nearby, but the code does not warn me that the peak has been discovered. I can see in SDR# that the peak are usually around -20dB or -12dB (i guess it depends on how close I am to the signal). 

When trying with a simple 446Mhz PMR radio on the 446 band it works perfectly. Really wish I could make it work with the Tetra signal as well!

**Comment by nootedandrooted** [comment]
Right. I will try my best to look into this in the following weeks. Currently i'm very busy with finishing my degree. I'm also interested in improving this project since it would be a useful tool for other projects as well :)

**Comment by laabak84** [comment]
Thanks a lot for your reply, I will be eagerly waiting :-) Good luck with your degree!
